scripts/apps/content-filters/services/ContentFiltersService.js

```diff
@@ -26,20 +26,12 @@ export function ContentFiltersService(api, $filter) {
         return api.remove(item);
     };
 
-    this.getAllFilterConditions = function(page, items) {
-        return _getAll('filter_conditions', page, items);
```

Comment:
not sure why not just put `api.getAll` here instead of modifying service api

Reply:
I initially did by modifying `ContentFiltersService` api actually, but as a second though it seemed better to me to use directly getAll as it is as simple. I can change it again no pb.

Reply:
changing the API is breaking current tests anyway, so I'll go back to first implementation.